test-flip-link-app/.cargo/config.toml

```diff
@@ -1,9 +1,9 @@
-[target.thumbv7m-none-eabi]
+[target.thumbv7em-none-eabi]
 runner = "qemu-system-arm -cpu cortex-m3 -machine lm3s6965evb -nographic -semihosting-config enable=on,target=native -kernel"
```

Comment:
The Cortex-M3 isn't an Armv7E-M device. You will need to change the emulated CPU type, and possibly the emulated board type (although maybe that will need new Rust drivers - not sure how much the test uses the chip)

Reply:
Mhh, it does execute fine for me locally 🤔

Reply:
It also works in CI: https://github.com/knurling-rs/flip-link/actions/runs/8988332696/job/24720052667#step:8:26

Reply:
According to [this table](https://en.wikipedia.org/wiki/ARM_Cortex-M#Instruction_sets) it should be either M4 or M7, correct?

Reply:
Switching to cortex-m4 works